pkg/appconsts/global_consts.go

```diff
@@ -67,3 +67,12 @@ func UpgradeHeightDelay() int64 {
 func HashLength() int {
 	return hashLength
 }
+
+// The following consts are not consensus breaking and will be applied straight after this binary is started.
+const (
+	// NonPFBTransactionCap is the maximum number of SDK messages, aside from PFBs, that a block can contain.
+	NonPFBTransactionCap = 200
+
+	// PFBTransactionCap is the maximum number of PFB messages a block can contain.
+	PFBTransactionCap = 600
+)
```

Comment:
[question][potentially blocking] I have a strong suspicion that we will need to modify these constraints in a future release. Should we make them versioned constants now to make it easier to modify them in future releases?

Reply:
versioning is a good idea, as we could increase the limit upon upgrading. 

since this is in prepare proposal and its not consensus breaking, this is ofc not required. the example listed here for versioning could be applied in that upgrade

opening a FLUP so we can :ship:

Reply:
#3973